Resources/Prototypes/Floof/Entities/Mobs/NPCs/xenopet.yml

```diff
@@ -0,0 +1,308 @@
+- type: entity
+  name: basic
+  id: MobSpaceFriend
+  parent: MobXenoRounyNPC
+  description: It looks friendly. Why don't you give it a hug?
+  components:
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: MovementAlwaysTouching
+  - type: MeleeWeapon
+    soundHit:
+        path: /Audio/Weapons/Xeno/alien_claw_flesh3.ogg
+    angle: 0
+    animation: WeaponArcClaw
+    damage:
+      types:
+        Blunt: 5
+        Slash: 5

```

Comment:
```suggestion
```
Avoid changing Xeno Default Attacks

Reply:
this is not a default xeno.  therefore it is not a default xeno attack. it's intended to be a possible ghost role. therefore it's tailored to be used by a player. I've strived to try and avoid changing values and parameters on the actual basegame xeno files so long as it did not involve fixing a bug (like the unrevivable bug)

Reply:
MeleeWeapon Prototype is already loaded by MobXenoRounyNPC, thats load RunnerNPC with the damage prototype.

This is again, undeeded.